.claude/epics/phase-2-enhanced-ux/github-mapping.md

```
@@ -0,0 +1,39 @@
+# GitHub Issue Mapping
+
+Epic: #11 - https://github.com/curdriceaurora/Local-File-Organizer/issues/11
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)